README.md

```diff
@@ -2,9 +2,25 @@ edX OAuth2 Provider [![Build Status](https://travis-ci.org/edx/edx-oauth2-provid
 ===================
 OAuth2 provider for edX Platform.
 
-`oauth2_provider` is a Django application used for authentication and authorization in the edx-platform. The
+`edx_oauth2_provider` is a Django application used for authentication and authorization in the edx-platform. The
 authentication mechanism is aligned with the OpenID Connect specification, supporting some of its basic features.
 
+Release notes for v1.0.0
+------------------------
+
+* `oauth2_provider` has been renamed to `edx_oauth2_provider`.  You will need to update your `INSTALLED_APPS` to use
+  the new name, and fix all imports.
+
+* If you are updating from `edx-oauth2-provider-0.5.9` or earlier, you will need to fake-apply the initial migration:
+
+      $ ./manage.py migrate --fake-initial
```

Comment:
It is helpful to know how the error manifests itself if someone forgets to fake-apply the migration.  Would they get a duplicate table SQL error?